source/_static/controllers/behringer_ddm4000_page6_microphone_channel.svg

```diff
@@ -0,0 +1,441 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="124.65pt" height="209.65pt" version="1.2" viewBox="0 0 124.65 209.65" xmlns="http://www.w3.org/2000/svg" xmlns:cc="http://creativecommons.org/ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:xlink="http://www.w3.org/1999/xlink">
+<metadata>
```

Comment:
Thanks, can you save the SVGs as optimized SVGs (Inkscape > Save As...) to reduce file size?

Reply:
I'm afraid they should already be optimized. I converted them from the manual (PDF) in inkscape and filtered them though https://www.svgminify.com/ afterwards. My inkspace version (1.0.2-2) offers either Inkscape SVG or plain SVG, both are ~20% bigger than the minified files.

Reply:
"Optimized SVG" is not available on Windows. I tried scour on Ubuntu with the recommended settings from the usage section, but the size doesn't change. Probably that website I used also uses scour internally.